### DISCUSSION TITLE
runtime activity

### DISCUSSION THREAD

**Discussion by GaiaBes** [Abe404/root_painter#16] [Q&A] [answered]
Hi! I just have one question. What happens if during the training the computer disconnects from the internet? 
If I connect it again to the internet and reconnect the runtime is it okay? Because it just happened to me and now everything seems to work again, the server is running and the epochs are loading. 
Thank you in advance

**Comment by Abe404** [comment, marked as answer]
Hi Gaia

Training keeps going until a stop training instruction is sent to the server (this can be done by clicking Network menu and then stop training option). It will also stop training if there are 60 epochs without progress. My recommendation is to leave it until the latter happens.

You can also stop training by quiting the server process (stopping the cell in google colab for example).

Disconnecting the client computer from the internet wont stop training.

> If I connect it again to the internet and reconnect the runtime is it okay?

Yes. I think it's fine to do that.

Kind regards,
Abraham
